pkgs/by-name/xr/xrsh/package.nix

```diff
@@ -0,0 +1,32 @@
+{
+  stdenv,
+  fetchgit,
+  lib,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "xrsh";
+  version = "0.142";
+
+  src = fetchgit {
+    fetchSubmodules = true;
+    url = "https://codeberg.org/xrsh/xrsh.git";
+    rev = "12f84e4cf72bf1e1268a6a4120594a825a8f66a7";
+    sha256 = "sha256-0MBBlqL2TWgN94ErI2Joub0Aub5XBxGWXILPDAfiNpc=";
+  };
+
+  dontBuild = true;
+
+  installPhase = ''
```

Comment:
Don't forget runHooks

Reply:
there are no runHooks

Reply:
https://nixos.org/manual/nixpkgs/stable/#sec-stdenv-phases

> When overriding a phase, for example `installPhase`, it is important to start with `runHook preInstall` and end it with `runHook postInstall`, otherwise `preInstall` and `postInstall` will not be run. Even if you don’t use them directly, it is good practice to do so anyways for downstream users who would want to add a `postInstall` by overriding your derivation.